src/components/Header/Header.style.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.header`
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 32px;
+`;
+
+export const Nav = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const Logo = styled.img`
+  width: 150px;
+`;
+
+export const Ul = styled.ul`
+  list-style: none;
+  display: flex;
+  align-items: center;
+`;
+
+export const Li = styled.li`
+  font-size: 16px;
+  margin-right: 25px;
```

Comment:
Out of curiosity, did you try out the perfect pixel technique? Iirc it's 25px in the design but should actually be 26px

Nothing to worry about, just curious as I said :smile:

Reply:
Yes 😄  I was getting the design measurements from figma. Should I have got it from zeplin? 🤔

Reply:
Ah no, the designs are the same. The point was more that the designs can't be trusted. It's not very important here but there seem to be a bit of spacing within the text in the designs. That's why I'm the CSS it requires 26px instead of 25px margin.

But honestly, that's a bit nitpicking. Nobody would check that in a real team 😉

Reply:
Ahh ok, I see what you mean 👍